chat.js

```diff
@@ -662,7 +662,7 @@ class CommandContext {
 			let lockType = (user.namelocked ? `namelocked` : user.locked ? `locked` : ``);
 			let lockExpiration = Punishments.checkLockExpiration(user.namelocked || user.locked);
 			if (room) {
-				if (lockType) {
+				if (lockType && !room.isHelp) {
 					this.errorReply(`You are ${lockType} and can't talk in chat. ${lockExpiration}`);
```

Comment:
Possibly I should mention that locked users can talk in help ticket rooms.

Reply:
And I should probably mention tickets in various other places (lock popups ect)

Reply:
Nah, those should happen later.